### DISCUSSION TITLE
I want to create a folder through the Setup file.

### DISCUSSION THREAD

**Discussion by jungjungdangdang** [community/community#47517] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hi

I am making a project in the c# environment. I'm almost done, but I got stuck making the last Setup file.As the title says, I want to create one folder through the Setup file. I want to make a setup file, That is same as Folders created by building projects. please tell me solution.

**Comment by unknowVariable** [comment, marked as answer]
There are a few different ways you can create a setup file that includes a folder in C#. One approach is to use a setup project in Visual Studio. Here are the general steps:

1. Open your C# project in Visual Studio.
2. Right-click on your solution and choose "Add" > "New Project".
3. Choose "Other Project Types" > "Setup and Deployment" > "Setup Project".
4. Give your setup project a name and click "OK".
5. Right-click on your setup project and choose "Add" > "Project Output".
6. In the "Add Project Output Group" dialog, select your C# project from the "Project" drop-down list, and choose "Content Files" as the "Output Type".
7. Click "OK".
8. Right-click on your setup project and choose "Add" > "Folder".
9. Give your folder a name.
10. Drag and drop any files or subfolders you want to include in the folder from the "File System Editor" onto the new folder.
11. Build your setup project (right-click on the project and choose "Build").
12. Your setup file will be located in the "bin/Release" (or "bin/Debug") folder of your setup project.

Another approach is to use a third-party setup creator tool, such as Inno Setup or Advanced Installer. These tools typically provide more advanced features and customization options than the built-in setup project in Visual Studio.

I hope this helps! Let me know if you have any further questions.

**Reply by jungjungdangdang** [reply]
Unfortunately, I've already solved the problem before you tell me know.Thank you for letting me know so well.
